.github/workflows/run_rc_validation.yml

```diff
@@ -530,4 +552,4 @@ jobs:
     if: always()
     steps:
       - name: Deleting Shared Pub Sub
-        run: gcloud pubsub topics delete --project=${USER_GCP_PROJECT}  ${{needs.generate_shared_pubsub.outputs.name}}
+        run: gcloud pubsub topics delete --project=${USER_GCP_PROJECT}  ${{needs.generate_shared_pubsub.outputs.name}}
```

Comment:
could you add a blank line at the end of the file please?

Reply:
Sure, I've added blank final lines to both files, thx for the catch.